tt_metal/impl/device/multi_device.hpp

```diff
@@ -52,8 +63,7 @@ class DeviceMesh
     void close_devices();
 
    private:
-    int num_rows;
-    int num_cols;
+    bool is_galaxy_;
```

Comment:
Let's not propagate this to any getters. We should get rid of these kind of bools.

Reply:
agreed, waiting on #10419 for removal 😄